dotnet-engine/Yggdrasil.Engine/YggdrasilEngine.cs

```diff
@@ -30,6 +57,8 @@ public void Dispose()
 
     public void TakeState(string json)
     {
+        customStrategies.MapFeatures(json);
```

Comment:
I think you should move this after taking the state and error handling.

Reply:
Good point!